src/client/Microsoft.Identity.Client/ITokenCacheInternal.cs

```diff
@@ -38,14 +39,7 @@ internal interface ITokenCacheInternal : ITokenCache, ITokenCacheSerializer
             AuthenticationRequestParameters requestParams,
             string familyId = null);
 
-        #endregion
-
-        #region For test
```

Comment:
There are other ways in which tests can get to the cache, no need for so many test hooks.